src/style_manager/view/PropertyIntegerView.js

```diff
@@ -5,7 +5,21 @@ const $ = Backbone.$;
 
 export default PropertyView.extend({
   templateInput() {
-    return '';
+    const pfx = this.pfx;
+    const ppfx = this.ppfx;
+    const icon = this.model.get('icon');
+    const textLabel = this.model.get('textLabel');
+    const iconEl = icon
+      ? `<i class="${icon}"></i>`
+      : textLabel
+      ? `<div>${textLabel}</div>`
+      : '';
```

Comment:
feel free to buck his coding style. likely eventually we will rewrite the whole thing lol